libsolidity/formal/SMTChecker.cpp

```diff
@@ -330,20 +330,35 @@ bool SMTChecker::visit(ForStatement const& _node)
 void SMTChecker::endVisit(VariableDeclarationStatement const& _varDecl)
 {
 	if (_varDecl.declarations().size() != 1)
-		m_errorReporter.warning(
-			_varDecl.location(),
-			"Assertion checker does not yet support such variable declarations."
-		);
-	else if (knownVariable(*_varDecl.declarations()[0]))
+	{
+		if (auto init = _varDecl.initialValue())
+		{
+			auto const& symbTuple = dynamic_pointer_cast<SymbolicTupleVariable>(m_expressions[init]);
```

Comment:
```suggestion
			auto symbTuple = dynamic_pointer_cast<SymbolicTupleVariable>(m_expressions[init]);
```
(I think the other version should also work, but better not confuse people - `dynamic_pointer_cast` returns a copy by value)